greeting.rb

```diff
@@ -0,0 +1,11 @@
+require 'sinatra/base'
+require './plant'
+
+class Greeting < Sinatra::Base
+	def self.connect(name)
```

Comment:
Great separation of classes. This is responsible for making the connection of the API and then taking back the filtered results.